docs/sources/installation.md

```diff
@@ -2,8 +2,72 @@
 
 Here are some requirements and guides to contributing to the Girl Develop It website code base. Be sure to review this when trying to install a local development environment! Be sure to reach out to us at [website@girldevelopit.com](mailto:website@girldevelopit.com) with questions.
 
+There are two ways to install the website on your local machine. We reccomend you use our Vagrant environment. This is a pre-packaged, configured development environment that includes a virtual machine. These instructions will work on Windows, Mac, or Linux machines.
+
 ---
 
+### How to Vagrant
+
+<!-- To set up the Vagrant environment locally:
+
+1. Install VirtualBox
+2. Install Vagrant
+3. clone the git repository
+4. `$ vagrant up`
+5. `$ vagrant ssh`
+6. `$ cd /opt/gdi/development`
+7. `$ bundle install`
+8. `$ rake db:create db:migrate db:seed`
+9. `$ bin/rails s`
+10. Point your browser to `localhost:3000`
+11. DEVELOP IT! \o/ -->
+
+#### 0. Install Git (Windows only)
+Git is a cross-platform tool that helps users collaborate on code projects. We use Git, combined with the site GitHub, to manage our website. Windows users will need to install Git on their machines. You can the latest version from [Git for Windows](https://git-for-windows.github.io/)
+
+Vagrant cannot run properly inside Powershell, so powershell-based tools like [GitHub Desktop](https://desktop.github.com/) won't work for Windows users.
+
+#### 1. Install VirtualBox
+VirtualBox is a cross-platform virtualization application (see: [the VirtualBox manual](https://www.virtualbox.org/manual/ch01.html)). That means it can install multiple virtual machines at the same time, which is great for development. You can download the latest version on [the VirtualBox website](https://www.virtualbox.org/wiki/Downloads).
+
+If you have an older version of VirtualBox on your machine, be sure to update it. Version 5.0.12 and below won't work with this setup.
+
+#### 2. Install Vagrant
+Vagrant is a wonderful tool to help you build complete development environments. Download the latest version on [the Vagrant website](https://www.vagrantup.com/downloads.html).
+
+#### 3. Clone the gdi-website Git repository
+Clone the Git repository into your local development folder: `git@github.com:girldevelopit/gdi-website.git`
+
+#### 4. Start Vagrant
+Depending on your machine, this could take a while.
+
+```
+$ vagrant up
+$ vagrant ssh
+```
+
+#### 6. Install and initialize the development environment
+
+Run the following:
+
+```
+$ cd /opt/gdi/development
+$ bundle install
+$ rake db:create db:migrate db:seed
+$ bin/rails s
+```
+
+Again, this could take several minutes to run.
+
+#### 7. View website
+Test your connection in a browser: `localhost:3000`
+
+---
+
+##Manual Installation
+
+If you prefer not to use Vagrant, you can manually install the site on your local machine. This instructions will only work for machines running Mac OS X or above.
```

Comment:
Still certain I got this working for PC (the Gemfile has conditions for OS types), but I can help add more details since there are things to comment out/change in the database.yml file if a person decides to go the manual route.